amazon-kinesis-client/src/main/java/software/amazon/kinesis/leases/LeaseManagementConfig.java

```diff
@@ -194,6 +194,13 @@ public class LeaseManagementConfig {
 
     private BillingMode billingMode = BillingMode.PAY_PER_REQUEST;
 
+    /**
+     * Whether to enabled deletion protection on the DyanmoDB lease table created by KCL.
+     *
+     * <p>Default value: false
+     */
+    private boolean deletionProtectionEnabled = false;
```

Comment:
Can we make this variable explicitly related to DynamoDB so as not to confuse customers, then we can track it easier in case there were ever more dependencies.

Reply:
Renamed variable